packages/website/src/pages/main/index.tsx

```diff
@@ -49,9 +57,16 @@ const Main = () => {
                               pathname === item ? styles.itemSelect : ""
                             }`}
                           >
-                            <Link to={`/${item}`} className={styles.itemLink}>
+                            <div
+                              className={styles.itemLink}
+                              onClick={() => {
+                                startTransition(() => {
```

Comment:
Wow, I was just about to do this in my next PR, using startTransition. i noticed that the preview page was no longer in the load state, turns out you've already done that! Awesome!